person/src/com/person/core/enumeration/ActionType.java

```diff
@@ -0,0 +1,11 @@
+package com.person.core.enumeration;
+
+public enum ActionType {
+    EAT,
```

Comment:
@tatevikb why you don't use id's and get By id method with throwing an exception

Reply:
@nsimonyan Done.